Parsers/ASimDns/Parsers/vimDnsMicrosoftNXlog.yaml

```diff
@@ -256,18 +313,12 @@ ParserQuery: |
         , DnsQueryType between (110 .. 248), 'Unassigned'
         , DnsQueryType between (261 .. 32767), 'Unassigned'
         , 'Unassigned'),
-       EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
+      EventResult=iff (EventResult == "Based on RCODE", iff(DnsResponseCode == 0, "Success", "Failure"),EventResult)
   | extend
     // Aliases
       IpAddr = SrcIpAddr,
-      Src = SrcIpAddr,
-    // Backward compatibility
-      Query = DnsQuery,
-      QueryType = DnsQueryType,
-      QueryTypeName = DnsQueryTypeName,
-      ResponseCode = DnsResponseCode,
-      ResponseCodeName = DnsResponseCodeName
+      Src = SrcIpAddr
   | project-away
       *_s, *_d, QTypeName, TenantId, SourceSystem, MG, ManagementGroupName, Computer, RawData
   };
-  ASimDnsMicrosoftNXLog (starttime, endtime, srcipaddr, domain_has_any, responsecodename, response_has_ipv4, response_has_any_prefix, eventtype, disabled)
+  ASimDnsMicrosoftNXLog (starttime=datetime(null), endtime=datetime(null), srcipaddr='', domain_has_any=dynamic([]), responsecodename='', response_has_ipv4='*', response_has_any_prefix=dynamic([]), eventtype='Query', disabled=false)
```

Comment:
please refer this parser for function call approach: from line 200-209
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimDns/Parsers/vimDnsInfobloxNIOS.yaml

Reply:
Sure thing, I will update it.